src/__tests__/load-modules.test.ts

```diff
@@ -176,7 +176,7 @@ describe('loadModules', function () {
       listModules: jest.fn(() => moduleLookupResult),
       require: jest.fn((path) => modules[path]),
     }
-    const opts: LoadModulesOptions = {
+    const opts: LoadModulesOptions<false> = {
```

Comment:
Is the generic arg needed since it specifies a default in the declaration?

Reply:
Ah, my bad, I had only specified the default in the `loadModules` function, not the type def itself.